src/firefly/js/visualize/WebPlotRequest.js

```diff
@@ -350,13 +350,15 @@ export class WebPlotRequest extends ServerRequest {
     /**
      *
      * @param wp
-     * @param {string} survey  must be one of 'j','h','k'
+     * @param {string} survey  must be one of 'asky', 'askyw', 'sx', 'sxw', 'cal'
+     * @param {string} band  must be one of 'j','h','k'
      * @param sizeInDeg less then .138 degrees (500 arcsec)
      * @return {WebPlotRequest}
      */
-    static make2MASSRequest(wp, survey, sizeInDeg) {
+    static make2MASSRequest(wp, survey, band, sizeInDeg) {
```

Comment:
There is a equivalent method in WebPlotRequest.**java**  that needs to be changed as well.  I think it is used by the FinderChart API.

Reply:
Thanks. I will double check in IFE but i think i refactor also the code there.